charts/ingress-monitor-controller-endpoint/values.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# EndpointMonitor
+#
+endpoint:
+  name: binbash.com.ar
```

Comment:
@lgallard I would suggest changing this to example.com, just to avoid cases where people wants to test this up using defaults.